packages/bpk-component-image/src/withLazyLoading.js

```diff
@@ -81,32 +109,40 @@ export default function withLazyLoading(Component, document) {
     // This function is taken from modernizr
     // See https://github.com/modernizr/modernizr
     // eslint-disable-next-line
-    supportsPassiveEvents() {
+    supportsPassiveEvents(): boolean {
       let supportsPassiveOption = false;
       try {
+        // $FlowFixMe
```

Comment:
The flow error is:
```
[flow] Object.defineProperty (Property not found in object literal)
```
I tried to the type annotation `:Object` didn't work

Reply:
Further to this as describer here https://github.com/facebook/flow/issues/1336 Flow doesn’t support getters and setters, there is a workaround, by adding `value` to the defined property, this bit of code however is from a third party library and I'm not keen to change it